qiskit_ibm_provider/ibm_backend.py

```diff
@@ -820,6 +820,10 @@ def _check_circuits_attributes(self, circuits: List[QuantumCircuit]) -> None:
             "https://qiskit.org/documentation/tutorials/circuits_advanced/05_pulse_gates.html` "
             "on how to use pulse gates."
         )
+        if len(circuits) > self._max_circuits:
+            raise IBMBackendValueError(
+                f"Number of circuits {len(circuits)} exceeds backend._max_circuits({self._max_circuits})"
```

Comment:
nit: maybe it's more clear if the error message is `...exceeds the the maximum for this backend, {}`

Reply:
Agree. Done.